lib/widgets/organization_list.dart

```diff
@@ -64,15 +64,15 @@ class OrganizationList extends StatelessWidget {
                 result.data!['organizationsConnection'] as List,
               );
             }
-            WidgetsBinding.instance.addPostFrameCallback((_) {
+
+            Timer(const Duration(seconds: 5), () {
```

Comment:
This should be a universal constant because the timeout will affect other functions too

Reply:
I am unclear about whether

Do you want me to set 5 seconds as global constant ?
or 
Do you want me to take the timer above orglist scope?

@palisadoes

Reply:
We are going to need a standard expiry timer for all similar transactions, not just orglist. Ideally I'd think it would need to be a constant.

Do you have an alternative solution?

Reply:
Sorry missed your message, yes this needs to be constant.
No for now i think this solution is sufficient